addons/website_sale_loyalty/views/portal_templates.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="portal_my_home_menu_loyalty" name="Portal layout : loyalty menu entries" inherit_id="portal.portal_breadcrumbs" priority="35">
+        <xpath expr="//ol[hasclass('o_portal_submenu')]/li[contains(@t-if,'sale_order')][2]" position="after">
+            <li t-if="page_name == 'loyalty'" t-attf-class="breadcrumb-item #{'active ' if not sale_order else ''}">
+                <a t-if="sale_order" t-attf-href="/my/loyalty">Loyalty</a>
+                <t t-else="">Loyalty</t>
+            </li>
+        </xpath>
+
+        <xpath expr="//ol[hasclass('o_portal_submenu')]/li[contains(@t-if,'sale_order')][1]" position="attributes">
+            <attribute name="t-if" add="(page_name != 'loyalty')" separator=" and "/>
+        </xpath>
+        <xpath expr="//ol[hasclass('o_portal_submenu')]/li[contains(@t-if,'sale_order')][2]" position="attributes">
+            <attribute name="t-if" add="(page_name != 'loyalty')" separator=" and "/>
```

Comment:
```
        <xpath expr="//t[@t-set='show_cart']" position="after">
            <t t-set="show_cart" t-value="show_cart and website_sale_order and website_sale_order.cart_quantity" />
        </xpath>
```

Reply:
Also, `page_name` was actually not populated anymore because the overridden method used to inject it had been removed.